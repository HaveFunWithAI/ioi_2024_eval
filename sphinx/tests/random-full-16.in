250 3000
218 218 100 131 20 221 88 100 229 229 221 175 43 221 130 88 43 229 88 221 218 100 175 221 131 71 183 183 17 183 175 218 131 100 17 183 100 17 160 88 229 184 229 130 184 160 17 184 229 229 175 218 218 71 131 218 160 131 20 43 184 43 218 229 175 218 20 184 183 43 160 20 100 131 71 183 175 183 221 218 229 88 229 20 20 175 218 131 221 20 43 183 221 131 20 43 20 160 43 218 184 71 160 175 184 100 71 17 229 88 184 183 131 130 130 184 184 43 71 184 131 17 218 229 221 221 183 88 131 43 160 229 160 131 183 221 43 183 175 160 43 71 100 71 20 160 130 130 183 175 221 229 43 131 43 160 17 20 184 221 218 88 160 131 175 88 218 229 131 221 218 221 218 130 131 131 184 229 88 221 88 221 160 221 130 100 100 17 175 229 130 184 175 71 43 100 160 184 88 160 88 17 160 131 88 218 130 130 160 71 131 20 218 221 100 17 17 130 221 20 17 71 100 88 88 175 221 100 229 175 100 218 218 43 218 100 221 218 175 43 160 175 71 131 100 131 184 17 43 183
45 73
27 45
73 164
45 67
82 164
27 230
45 135
67 72
164 180
135 227
89 230
89 246
8 164
214 246
111 227
86 214
39 89
43 164
14 180
155 230
14 167
54 155
10 27
39 199
17 86
21 246
39 168
111 148
180 217
189 214
27 242
8 119
45 145
155 166
43 78
43 47
168 175
69 242
175 204
82 128
2 166
67 154
175 176
82 163
199 218
4 246
124 176
58 246
32 72
5 21
49 111
145 172
17 238
26 58
14 183
72 142
77 180
2 159
77 210
89 195
111 141
82 127
4 197
25 128
79 145
109 204
167 208
168 192
134 172
67 186
108 192
140 218
208 220
156 176
8 221
145 177
2 185
3 8
107 220
52 176
79 85
112 238
20 218
18 47
154 229
67 104
4 245
168 237
79 248
183 194
127 216
154 184
154 228
104 120
108 151
32 173
199 226
110 185
7 145
130 204
20 68
10 174
27 182
35 155
135 181
5 243
49 179
134 241
17 200
183 239
1 35
54 70
124 146
22 77
132 245
24 159
186 249
72 91
129 192
146 160
32 60
33 159
49 190
58 232
13 238
51 177
105 214
139 166
178 195
12 186
99 181
62 204
14 123
67 118
222 229
87 228
9 140
184 212
21 193
101 214
93 182
88 174
93 191
1 136
143 193
194 247
120 138
149 208
64 109
67 188
102 172
93 240
189 201
117 243
113 146
50 190
102 170
109 244
167 205
150 188
39 165
97 170
35 40
50 187
52 61
12 131
130 137
34 82
76 170
93 161
18 209
16 146
46 149
184 235
73 157
57 204
140 196
56 209
198 228
66 72
36 50
162 229
23 192
74 197
81 173
31 54
92 194
126 149
59 249
28 212
202 227
171 188
94 194
0 92
90 242
81 225
131 234
172 223
103 191
63 218
41 163
96 184
115 205
56 133
54 80
100 221
46 236
15 22
98 235
125 135
48 228
54 144
84 189
114 205
169 209
53 108
5 37
58 75
33 106
159 215
34 213
65 201
31 55
100 233
11 245
213 224
48 152
114 206
18 158
44 110
6 217
27 147
26 121
63 153
219 236
83 85
116 224
207 249
36 231
42 54
13 29
150 211
43 95
71 128
2 122
38 59
13 19
59 203
30 153
26 245
34 87
95 129
2 177
122 129
27 157
94 184
111 150
63 87
3 28
24 108
35 194
51 115
82 178
95 162
110 154
106 121
10 225
16 130
41 46
117 186
68 124
5 73
79 195
24 63
223 233
69 107
43 241
1 6
19 63
75 216
111 206
11 170
142 240
50 67
115 120
54 247
220 248
7 118
82 229
92 127
134 222
122 157
3 121
17 162
132 201
110 240
112 205
90 124
60 148
126 226
160 202
51 222
26 84
84 224
41 58
47 127
65 75
106 180
119 138
136 244
59 151
29 204
56 208
118 131
76 160
87 238
155 163
18 239
16 182
83 181
66 113
25 110
179 221
65 147
41 191
135 177
82 242
122 123
131 231
15 124
54 218
32 132
4 91
182 194
108 198
123 234
76 219
116 131
10 228
207 227
3 70
83 148
175 205
9 194
12 44
107 174
5 127
158 183
60 199
204 225
151 200
103 124
50 229
22 235
130 232
165 199
183 207
39 119
136 231
83 188
37 113
39 90
16 227
81 87
53 190
72 84
159 189
3 168
1 125
5 75
68 149
38 214
38 159
61 203
21 126
3 11
19 218
71 194
63 240
121 175
20 100
56 198
208 214
158 194
82 218
121 221
74 108
146 242
207 214
35 190
54 63
53 184
163 213
44 75
98 146
155 233
8 88
53 145
138 202
104 228
233 247
181 214
118 161
68 118
81 184
221 247
64 243
44 205
9 13
58 67
12 205
154 240
110 170
0 90
116 205
52 126
55 235
70 176
135 223
153 228
190 208
106 219
177 214
58 235
50 60
7 80
158 171
83 133
22 173
164 241
185 241
10 73
124 125
175 199
53 137
114 132
54 220
30 216
190 223
44 157
126 203
51 174
102 123
42 215
9 162
178 227
1 151
172 221
19 38
12 179
149 183
8 23
50 55
0 228
30 122
74 110
8 142
56 72
6 66
60 79
98 161
104 237
170 213
6 37
18 57
38 194
93 175
79 170
79 244
232 246
86 197
128 228
87 177
154 180
170 177
186 194
23 179
99 154
158 163
9 113
101 217
9 25
36 244
55 92
54 92
65 169
135 214
23 249
72 168
124 248
8 107
181 242
83 246
56 86
103 180
46 136
81 196
196 229
19 238
93 190
60 172
86 96
183 235
43 129
62 145
116 206
72 154
118 195
69 78
197 201
71 154
120 133
34 137
60 154
1 226
67 175
61 163
125 188
134 232
18 163
154 209
155 186
39 137
37 188
139 199
155 227
96 118
26 239
134 167
99 179
27 32
197 208
67 173
41 209
22 161
1 7
8 169
151 237
11 220
31 122
24 195
59 118
69 104
91 152
10 145
94 155
99 213
44 209
152 161
36 178
119 155
93 107
71 224
96 226
87 210
26 125
62 197
18 85
78 202
51 53
131 137
18 147
32 169
54 166
100 104
28 33
191 239
109 166
34 126
0 202
193 215
48 112
118 155
104 114
10 176
147 242
159 225
110 123
56 142
204 212
155 207
69 239
71 173
52 212
13 194
40 97
147 230
1 86
23 60
14 190
53 170
46 168
10 207
145 243
109 203
70 119
48 165
146 196
158 176
90 147
35 232
123 207
4 170
32 179
70 138
7 37
12 64
141 200
167 173
185 187
119 145
8 33
15 19
177 190
133 193
19 121
153 236
45 243
100 113
229 246
71 129
167 198
106 238
159 236
28 191
16 154
62 165
19 137
2 134
11 40
98 190
59 196
102 127
80 247
96 117
68 194
41 154
1 209
53 211
91 190
15 195
45 218
176 247
137 176
67 113
76 133
197 224
116 162
9 45
193 219
83 110
8 152
144 218
37 122
112 239
144 190
14 46
149 245
60 152
86 111
48 196
136 239
24 196
42 97
178 244
169 222
37 94
104 185
36 90
63 84
134 212
27 129
26 108
12 132
119 197
74 249
86 245
148 162
85 165
50 144
66 180
159 168
115 226
61 141
67 242
6 59
37 90
106 158
56 92
55 238
14 248
47 104
53 180
172 232
54 90
118 197
32 39
38 242
60 242
174 181
16 172
149 198
86 137
65 189
23 189
115 239
36 249
144 178
122 154
26 177
38 41
7 130
48 216
134 246
65 109
113 234
24 199
147 236
173 221
6 92
2 135
19 95
172 225
82 154
19 227
96 202
20 201
42 237
191 248
113 188
2 111
54 215
59 113
103 167
112 144
28 101
15 118
174 244
30 67
147 219
113 140
125 130
115 121
27 58
97 188
79 235
166 229
62 150
54 195
110 171
69 214
27 197
157 202
79 138
30 220
19 195
44 116
58 133
128 141
132 170
126 210
65 156
88 115
0 42
84 144
117 238
139 183
68 78
101 136
73 175
63 137
90 113
40 57
106 149
38 64
122 140
20 155
5 211
85 134
164 220
140 240
124 168
152 236
101 106
82 233
15 197
19 204
163 183
29 114
31 119
114 163
208 233
164 213
170 193
31 75
152 220
180 231
225 237
56 87
60 226
131 240
19 44
57 192
16 91
175 226
50 91
76 164
45 80
150 212
140 215
62 116
71 181
93 95
52 135
231 240
51 164
58 110
35 109
24 124
67 87
108 173
89 130
136 207
110 130
27 217
31 56
14 40
122 247
1 2
39 112
94 123
39 67
62 222
119 127
116 152
156 223
27 199
176 192
29 210
150 157
93 174
32 219
56 197
21 132
17 112
114 149
161 233
219 233
41 77
59 222
228 237
33 132
62 172
26 196
65 221
27 218
30 58
41 143
42 93
83 144
169 201
17 121
12 63
215 245
152 200
146 205
89 116
25 171
179 205
92 109
10 135
153 239
43 234
78 81
49 158
121 129
61 219
117 242
37 111
142 149
127 210
5 60
110 236
121 186
26 151
32 146
126 189
16 152
70 90
90 198
190 227
157 217
63 104
71 164
60 177
124 179
42 73
173 233
128 162
23 232
3 4
80 152
46 240
19 159
84 246
135 172
80 104
68 163
114 201
83 243
14 247
44 176
71 76
94 141
35 174
5 20
147 154
7 217
30 224
63 111
81 113
45 193
124 161
32 61
132 150
15 206
35 126
55 107
17 118
19 141
31 85
23 80
27 142
61 178
64 187
181 199
68 213
16 79
37 224
146 166
104 163
61 80
0 80
15 221
123 231
26 65
210 239
13 246
63 229
80 144
99 237
63 189
64 153
6 231
69 237
141 171
97 149
95 209
58 205
121 201
44 181
148 222
67 85
85 214
57 92
66 233
217 223
33 181
139 229
47 92
99 188
61 69
110 206
76 201
104 148
65 176
136 217
94 163
43 166
17 190
96 128
33 157
2 128
24 182
64 240
118 219
128 233
165 227
110 218
125 249
39 68
12 51
58 218
166 217
60 240
146 241
48 160
142 150
143 246
167 178
58 173
29 211
79 108
75 195
28 46
125 231
150 160
67 115
8 239
4 198
203 213
111 163
81 138
163 218
37 144
157 161
94 144
22 72
207 233
104 235
210 228
23 39
7 114
98 105
15 24
20 51
226 243
156 242
31 90
118 213
42 146
45 199
108 233
42 98
109 193
115 156
43 194
18 117
23 196
57 78
68 130
145 152
40 123
153 168
49 104
113 222
2 43
18 178
55 163
110 137
0 169
14 219
62 203
64 157
180 219
203 230
22 94
106 108
10 182
114 135
10 202
8 230
147 241
79 198
45 117
116 165
95 109
105 239
167 200
17 132
13 87
10 120
16 220
79 112
109 157
14 56
69 220
8 241
17 36
157 194
135 143
66 208
165 229
135 249
1 211
55 217
99 192
54 102
39 224
53 118
83 199
12 213
146 175
52 172
45 183
133 235
127 211
73 120
10 70
229 248
149 170
76 203
5 158
85 135
27 87
32 154
129 148
123 216
97 141
134 186
13 178
85 112
11 140
77 91
109 196
56 115
48 225
146 154
82 236
179 190
129 186
80 115
55 64
119 153
95 116
168 194
70 236
141 248
21 47
89 153
62 120
87 103
110 225
38 236
118 211
205 233
150 184
12 236
50 160
134 163
129 197
15 121
88 236
93 132
29 213
5 242
74 232
176 224
34 166
190 206
176 248
48 190
59 137
31 153
154 238
52 67
5 180
76 92
125 207
79 219
86 149
16 24
86 204
26 43
3 91
216 223
112 193
190 237
76 249
88 247
158 217
122 133
228 249
198 207
242 245
65 116
185 228
75 83
62 113
3 181
39 53
2 102
8 70
181 218
73 233
40 53
187 209
48 138
21 153
21 208
166 226
231 238
27 102
70 83
32 195
72 125
53 185
87 186
200 240
7 230
9 107
70 211
222 225
7 89
32 101
137 189
22 220
53 74
67 76
128 156
151 173
16 83
6 118
39 177
106 130
134 209
90 223
58 233
46 114
92 201
148 225
69 118
101 105
117 153
76 117
35 136
106 159
6 155
95 136
147 209
18 30
27 237
206 222
58 96
7 246
75 157
71 169
113 243
221 245
25 201
174 219
17 30
41 94
164 236
27 160
44 69
65 210
72 235
104 155
10 19
124 175
201 219
82 121
34 247
222 248
29 190
43 172
47 61
147 167
161 249
101 234
26 241
125 151
57 88
80 158
192 220
226 240
62 242
47 103
26 80
15 148
50 64
42 116
84 203
100 167
84 243
6 201
5 131
164 209
32 113
143 173
76 153
6 46
204 214
0 172
22 99
154 189
12 182
9 237
87 99
144 150
134 148
50 186
113 169
77 143
157 227
36 234
127 155
81 181
122 202
78 146
121 178
32 139
48 88
29 212
91 171
47 172
109 156
97 168
66 183
140 143
82 169
64 200
23 112
84 192
98 209
217 241
102 162
38 62
40 233
1 162
199 227
11 70
53 72
173 182
110 245
52 165
54 96
0 98
123 196
25 222
27 138
44 143
103 163
136 168
66 123
50 145
115 172
147 189
0 141
199 201
140 148
18 46
207 228
5 117
96 111
6 57
24 109
184 213
203 233
118 225
63 193
116 148
118 137
149 210
240 243
0 58
64 214
30 98
221 227
142 221
65 148
32 172
8 149
121 236
59 99
87 96
72 85
68 76
46 72
84 183
142 158
93 215
17 55
141 240
31 84
65 157
222 237
77 224
110 166
25 52
138 248
31 221
51 239
2 183
18 243
138 145
27 248
44 109
46 233
11 32
43 90
97 148
131 214
177 178
103 204
45 122
37 190
76 239
14 69
1 65
110 228
14 98
11 87
196 232
86 219
3 233
63 133
78 82
60 232
158 200
90 120
108 225
21 146
105 210
163 168
23 237
59 202
29 189
76 182
87 226
81 94
104 189
83 186
96 140
71 161
14 111
55 210
158 219
32 196
2 117
132 236
47 208
21 102
109 248
38 82
22 193
111 219
52 117
59 96
11 194
35 160
53 146
28 165
146 239
76 91
76 248
82 241
156 215
76 95
173 246
83 206
55 185
184 229
35 184
227 236
66 206
5 99
35 242
78 187
95 217
26 200
112 155
52 163
46 61
151 224
112 231
39 243
178 204
55 123
106 225
6 178
53 67
93 189
77 203
14 131
148 177
17 26
55 129
32 214
16 95
41 72
42 249
21 90
32 48
99 108
168 184
31 59
11 110
1 55
136 144
47 176
141 210
26 180
27 158
193 238
225 247
32 188
10 165
208 242
124 207
36 194
87 97
92 217
0 247
185 234
84 114
39 219
160 190
26 143
146 164
36 224
111 114
1 244
191 194
133 186
13 20
148 191
75 115
112 172
51 197
40 170
99 107
65 135
167 188
138 240
110 201
0 204
126 233
19 135
66 141
184 200
61 241
12 169
35 103
105 245
187 215
75 89
36 191
136 205
46 178
26 29
28 164
30 112
78 181
38 241
172 234
215 229
95 206
106 198
187 247
1 241
18 150
78 204
72 121
39 148
32 226
177 248
46 216
74 107
122 188
48 213
58 198
49 175
77 118
105 176
39 97
50 167
138 198
54 118
99 159
166 207
121 210
31 214
37 126
2 16
92 116
95 101
112 148
132 193
65 130
63 168
144 244
76 181
129 216
104 168
55 90
96 150
208 240
18 62
158 227
162 168
179 187
10 243
31 144
37 160
50 69
20 98
29 225
60 139
31 89
80 187
31 106
3 228
70 120
55 114
16 104
32 63
62 234
100 214
34 181
207 246
235 243
108 216
109 184
21 96
167 192
15 140
22 58
25 74
106 236
7 195
144 156
89 178
78 223
59 235
101 180
41 132
8 109
223 247
169 192
87 178
80 148
151 231
46 129
85 204
83 135
135 201
242 246
102 124
39 41
60 189
180 241
213 244
38 219
77 109
159 246
50 178
0 178
133 159
122 183
93 157
138 204
45 228
181 227
25 34
179 216
99 127
178 221
133 181
155 243
113 127
35 180
35 139
71 229
59 214
55 228
45 139
0 158
24 84
51 170
189 243
166 235
60 99
213 248
131 247
39 205
194 222
185 192
101 144
2 55
191 245
202 226
90 192
129 142
9 247
146 187
23 162
78 182
190 224
123 149
133 203
128 244
13 153
7 103
44 53
168 217
3 84
49 217
9 31
42 68
12 185
148 232
62 135
140 177
179 201
128 135
34 157
45 223
81 157
70 132
38 222
68 154
14 181
38 248
32 228
60 247
98 152
54 136
9 18
4 85
47 209
166 178
12 235
16 167
56 190
127 149
179 238
88 129
2 107
134 237
13 230
95 229
41 179
165 188
60 217
108 215
65 173
4 174
6 198
11 244
30 94
48 120
133 187
22 249
50 146
47 64
208 211
29 171
17 212
205 218
60 71
113 159
90 151
73 148
8 50
108 197
87 208
34 67
1 173
140 214
18 174
105 242
38 139
67 160
138 153
45 83
111 128
169 241
130 135
77 225
87 215
96 239
67 68
46 231
28 132
51 122
134 161
70 154
123 248
122 195
115 242
123 193
39 195
200 201
61 129
61 133
11 213
89 214
239 249
25 224
201 215
239 244
38 66
74 233
8 41
169 200
189 235
133 201
39 130
0 54
19 25
14 99
174 246
72 245
44 102
44 120
36 166
138 196
103 211
77 215
217 249
94 100
2 64
14 22
0 222
68 249
68 122
128 205
156 227
34 208
158 164
79 208
25 198
49 145
7 119
46 152
40 214
110 193
70 203
207 235
17 166
84 105
26 181
0 104
74 155
52 68
111 229
131 178
34 47
161 235
48 114
162 183
22 118
74 161
37 161
35 202
56 106
182 226
142 147
104 123
151 174
57 95
28 53
9 169
49 208
5 34
9 120
67 153
65 248
16 85
183 241
34 162
52 121
31 215
52 182
74 242
79 150
193 235
131 236
132 155
10 66
149 160
130 245
76 132
161 217
122 239
35 38
106 249
128 202
68 228
29 158
85 229
60 65
41 127
27 233
169 171
117 241
62 94
81 84
143 210
40 99
139 195
37 200
54 163
35 129
79 234
83 212
0 128
114 200
96 238
100 218
3 55
74 188
49 188
32 232
94 157
105 203
90 133
201 235
80 214
16 183
133 244
32 175
65 119
195 227
32 248
125 236
0 139
122 197
167 226
134 230
18 210
19 246
63 112
86 102
52 245
3 243
156 169
37 206
111 161
29 106
141 155
200 246
17 81
33 66
5 177
107 195
116 187
136 229
115 234
177 225
82 149
142 230
121 140
35 146
212 224
86 160
115 203
74 116
109 226
76 179
35 65
173 198
186 247
34 75
33 179
130 170
56 57
46 140
6 21
186 201
233 239
228 243
28 178
8 25
32 35
43 236
198 211
72 102
183 220
72 177
5 69
63 120
72 139
172 180
204 224
21 38
23 185
172 188
232 238
35 148
4 94
162 193
86 230
42 194
2 241
32 50
9 145
16 148
109 222
167 246
175 189
5 195
0 27
195 223
27 91
102 190
10 85
7 66
217 242
34 215
7 86
192 227
46 141
132 233
106 138
156 191
21 211
195 238
209 247
200 202
91 168
209 224
26 93
16 76
66 117
93 193
82 109
1 231
2 138
87 227
83 100
31 101
107 135
136 214
119 183
139 238
70 85
10 155
72 215
203 247
176 201
164 171
59 81
161 214
25 86
100 146
96 209
12 183
129 137
38 110
97 127
89 92
15 26
182 197
112 219
42 105
32 116
191 249
141 231
104 206
98 193
125 217
41 65
41 221
16 181
93 183
92 205
88 212
13 69
10 40
107 211
139 237
52 78
145 174
31 123
0 10
29 101
65 212
36 157
25 167
30 157
7 138
4 114
53 93
17 68
220 247
135 234
44 153
31 165
95 214
14 218
8 157
137 201
98 168
64 142
25 143
159 220
196 242
76 245
95 234
53 96
21 106
21 240
113 192
76 115
23 215
23 199
138 218
163 170
73 180
190 225
161 203
59 103
40 122
180 248
68 198
123 215
161 170
112 176
53 58
4 87
35 163
73 144
156 171
2 40
156 236
57 157
1 234
38 217
185 246
76 99
41 216
130 156
97 221
201 239
37 187
42 61
41 99
76 180
132 246
13 147
114 145
118 136
151 176
38 117
174 215
186 210
126 249
90 121
82 101
3 173
161 189
91 102
47 175
36 103
50 58
168 183
76 196
17 204
14 177
66 136
39 203
32 78
3 95
114 127
4 173
45 97
37 101
20 153
42 81
87 130
97 225
135 169
28 182
65 115
126 224
51 247
29 94
115 127
115 192
35 81
10 192
36 161
106 217
9 243
2 243
232 241
108 111
124 210
182 199
95 118
64 212
64 236
62 215
198 212
146 201
90 209
60 213
55 87
44 95
30 178
33 186
17 237
51 52
115 215
35 161
80 167
49 211
125 202
143 192
27 77
33 176
70 97
63 226
111 222
15 231
57 72
56 137
96 138
55 67
1 191
188 201
36 216
51 105
20 236
38 147
131 210
42 204
125 193
72 113
101 212
27 132
22 92
61 161
102 157
153 238
88 233
232 249
172 216
102 128
69 114
166 213
14 214
34 100
189 213
34 194
127 196
8 21
18 122
114 183
106 191
67 103
103 130
93 94
78 238
4 117
174 223
137 239
180 222
37 54
106 144
114 182
96 208
70 232
24 122
32 171
62 74
165 180
165 209
71 123
165 201
75 182
32 134
5 236
5 58
84 185
231 247
126 214
178 179
64 210
24 64
6 146
120 195
13 174
22 97
116 184
15 216
120 168
112 187
63 196
4 36
116 232
34 61
80 241
80 95
64 223
135 224
103 133
139 202
191 205
222 232
40 197
130 168
100 226
52 184
86 186
228 229
132 244
43 66
32 137
61 115
23 82
113 116
69 213
76 229
147 246
62 247
65 153
118 193
85 241
64 176
96 135
123 142
98 212
58 191
50 115
34 212
135 196
147 210
62 199
67 184
136 171
95 131
98 106
8 20
65 106
61 76
8 238
68 233
90 161
114 141
133 180
144 147
15 16
32 201
113 203
52 131
6 208
144 202
44 87
126 166
131 147
46 134
197 219
16 242
49 138
0 184
175 231
27 229
111 125
13 195
2 94
42 104
14 141
52 191
44 97
169 246
25 87
175 179
83 106
18 225
37 137
46 207
5 107
106 166
44 89
129 246
56 114
78 156
178 226
44 73
38 191
15 238
161 219
48 50
15 224
6 10
179 234
36 84
175 243
118 181
43 67
18 119
48 115
179 244
94 96
190 195
4 158
83 179
24 59
213 235
30 183
208 232
200 229
19 173
83 162
89 165
187 238
181 225
60 179
87 98
85 89
31 42
191 204
122 209
88 186
84 247
76 224
188 229
104 179
156 175
162 188
23 107
168 221
97 117
109 142
34 58
67 143
62 226
30 246
107 162
78 221
50 54
4 201
215 238
31 201
145 218
40 212
96 242
25 183
200 244
158 165
40 115
90 148
96 187
85 195
89 212
49 148
234 240
7 141
37 231
87 229
6 106
155 229
61 65
50 214
165 168
132 205
35 52
21 130
106 170
98 234
155 231
70 201
2 124
156 248
204 207
184 199
28 193
197 239
88 149
72 203
160 247
103 148
51 202
25 75
8 185
140 241
193 207
69 235
28 141
41 129
105 237
183 225
80 194
1 51
145 223
15 63
124 190
40 83
65 216
85 197
115 243
103 237
117 137
136 204
84 111
90 182
22 219
12 165
108 165
197 241
20 49
28 195
67 131
151 238
17 54
103 177
29 238
25 158
40 211
64 145
152 205
26 209
34 216
149 241
108 145
46 212
190 222
56 151
124 227
83 88
32 187
5 156
2 60
215 222
44 247
112 195
52 137
143 195
136 146
43 162
22 202
155 235
145 148
9 117
173 244
85 131
45 180
43 48
28 241
29 129
162 174
86 232
102 111
59 181
59 69
69 221
33 69
61 156
63 163
7 90
9 218
137 161
9 146
109 210
155 214
37 88
0 14
33 171
77 158
192 203
97 106
115 206
192 234
49 177
7 51
46 185
51 220
101 137
46 146
10 46
6 168
41 64
15 210
61 138
44 202
44 175
20 69
58 249
169 187
100 240
101 238
47 193
64 163
33 143
115 158
2 187
82 210
13 211
28 247
23 209
130 162
206 229
0 155
165 181
23 186
17 83
203 249
46 78
44 207
28 146
138 207
13 83
90 244
13 159
51 140
78 99
45 61
137 157
13 109
27 226
7 115
173 187
12 91
45 186
85 218
14 222
33 217
89 210
15 230
108 144
63 236
55 166
51 80
172 199
43 170
222 249
147 224
215 240
154 225
107 156
155 204
50 135
48 68
98 171
56 76
199 215
45 198
21 158
27 224
16 107
165 189
144 249
31 39
172 200
15 85
137 188
3 142
25 109
78 104
41 213
94 138
153 210
16 195
8 179
0 137
227 237
33 67
16 249
23 36
2 155
43 183
119 122
48 179
81 142
150 225
33 229
110 223
42 157
71 117
14 118
141 166
170 237
40 124
163 247
110 221
52 167
225 238
63 162
28 154
43 237
66 249
0 200
148 213
44 159
216 247
215 233
57 122
99 117
108 240
66 121
42 52
15 105
50 83
53 154
37 132
222 231
6 48
16 90
65 99
208 236
47 66
96 100
61 223
63 221
3 196
56 129
95 124
139 187
73 135
1 11
237 249
73 232
67 127
103 217
145 203
181 216
97 134
86 222
22 39
6 211
68 102
120 186
19 231
128 154
189 246
70 217
213 220
209 218
2 222
113 229
103 194
18 127
102 181
12 184
190 220
69 137
171 231
22 131
98 206
17 231
195 225
200 211
95 172
36 117
22 128
7 175
58 114
0 148
84 245
1 3
23 133
61 113
186 232
103 153
45 56
60 235
193 228
80 199
123 235
54 133
81 93
69 205
14 148
80 218
184 198
103 238
157 182
148 247
27 44
30 155
181 201
21 234
58 82
4 71
126 234
56 206
118 239
105 228
128 130
91 129
107 181
166 168
3 184
120 146
36 57
40 242
59 70
66 85
27 46
25 93
45 109
40 90
158 190
115 130
162 241
29 133
154 176
157 196
84 210
54 161
53 144
94 135
7 143
185 188
74 217
25 148
35 196
140 223
0 193
32 215
40 223
84 115
135 184
116 154
11 228
76 211
35 135
29 202
106 141
102 147
105 192
224 225
77 234
116 146
61 172
99 195
159 203
158 161
118 134
30 124
45 184
59 145
43 144
217 231
36 88
28 143
15 31
135 160
61 74
100 194
9 211
17 243
12 60
200 208
74 209
234 245
53 209
200 233
170 208
5 86
44 169
102 143
125 165
138 209
187 221
101 132
29 155
56 178
99 162
112 215
209 246
179 212
129 174
187 207
118 126
110 191
78 170
41 51
121 164
243 246
86 168
144 210
84 219
140 206
136 188
12 107
9 88
111 216
32 249
28 174
100 171
183 210
168 173
237 246
101 218
54 116
83 120
135 232
234 237
88 134
117 249
142 162
87 153
2 133
11 111
15 127
13 243
160 184
154 244
71 212
20 127
25 88
87 124
184 244
57 87
98 244
170 243
93 184
109 207
109 202
169 217
114 245
127 187
144 145
34 144
116 118
82 93
100 107
81 244
22 69
226 230
111 139
196 238
15 213
129 134
23 121
69 109
173 230
16 84
0 205
71 97
27 98
95 152
209 217
18 156
54 129
36 167
117 126
12 221
180 182
170 205
79 188
84 87
2 164
11 236
103 114
220 239
122 226
32 107
15 172
91 159
19 130
21 39
221 236
83 193
89 234
84 194
160 248
32 119
114 208
92 143
107 191
101 208
17 63
170 214
154 158
110 239
30 38
64 238
42 198
33 191
121 153
67 205
101 201
184 186
96 245
3 64
94 220
118 241
0 150
123 175
120 147
10 171
177 234
88 202
42 162
11 225
70 142
67 241
27 223
63 125
176 178
55 61
241 244
62 86
187 197
160 171
127 192
171 209
164 174
29 160
75 169
14 106
10 109
206 228
36 202
