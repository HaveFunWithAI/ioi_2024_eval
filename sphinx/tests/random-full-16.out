250 2248
218 218 100 131 20 221 88 100 229 229 221 175 43 221 130 88 43 229 88 221 218 100 175 221 131 71 183 183 17 183 175 218 131 100 17 183 100 17 160 88 229 184 229 130 184 160 17 184 229 229 175 218 218 71 131 218 160 131 20 43 184 43 218 229 175 218 20 184 183 43 160 20 100 131 71 183 175 183 221 218 229 88 229 20 20 175 218 131 221 20 43 183 221 131 20 43 20 160 43 218 184 71 160 175 184 100 71 17 229 88 184 183 131 130 130 184 184 43 71 184 131 17 218 229 221 221 183 88 131 43 160 229 160 131 183 221 43 183 175 160 43 71 100 71 20 160 130 130 183 175 221 229 43 131 43 160 17 20 184 221 218 88 160 131 175 88 218 229 131 221 218 221 218 130 131 131 184 229 88 221 88 221 160 221 130 100 100 17 175 229 130 184 175 71 43 100 160 184 88 160 88 17 160 131 88 218 130 130 160 71 131 20 218 221 100 17 17 130 221 20 17 71 100 88 88 175 221 100 229 175 100 218 218 43 218 100 221 218 175 43 160 175 71 131 100 131 184 17 43 183
